protocol/data.proto

```diff
@@ -125,6 +125,36 @@ message TimeRecord {
   optional double sensor = 2;
 };
 
+
+///////////////////////////////////////////////////////////////////////////////
+/// The following describes the range-Doppler stream
+///////////////////////////////////////////////////////////////////////////////
+message LinearSamplingParams {
```

Comment:
Can we add a bit comments on this? I am not sure this is obvious (even to me).

Reply:
I've added comments to make it more clear. Let me know if you think It requires more description.